front/src/applications/stdcm/hooks/useStdcmConsist.ts

```diff
@@ -47,6 +61,12 @@ const useStdcmConsist = () => {
     dispatch(updateMaxSpeed(totalMaxSpeed === 0 ? undefined : totalMaxSpeed));
   };
 
+  const loadingGauge = useSelector(getLoadingGauge);
+  const onLoadingGaugeChange = (option?: LoadingGaugeType) => {
```

Comment:
If you added the type in the previous comment.
Also we always have to select one of the options so they can't be undefined
```suggestion
  const onLoadingGaugeChange = (option: STDCMLoadingGauge) => {
```